spec/controllers/announcements_controller_spec.rb

```diff
@@ -15,6 +15,23 @@
     @app_config = FactoryGirl.create(:app_config)
   end
 
+  shared_examples 'can hide announcement' do
+    before do
+      @announcement = FactoryGirl.create(:announcement)
+      @attributes = FactoryGirl.attributes_for(:announcement)
+      request.env['HTTP_REFERER'] = 'where_i_came_from'
+      get :hide, id: @announcement
+    end
+    it 'should set some cookie values' do
+      verifier = ActiveSupport::MessageVerifier.new(
+        Reservations::Application.config.secret_key_base)
+      name = 'hidden_announcement_ids'
+      @request.cookies[name] = verifier.generate([@announcement[:id].to_s])
```

Comment:
Why are we using `@request` instead of just `request`? Changing it doesn't appear to break the spec.